src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependencyNodeId.cs

```diff
@@ -133,7 +133,7 @@ public static DependencyNodeId FromString(string serializedId)
 
         public override int GetHashCode()
         {
-            return unchecked(ToString().GetHashCode());
+            return unchecked(ToString().ToLowerInvariant().GetHashCode());
```

Comment:
Most correct to match Equals precisely would be `StringComparer.OrdinalIgnoreCase.GetHashCode(ToString());` OrdinalIgnoreCase is equivalent to ToUpperInvariant of each char + compare, not ToLowerInvariant, and IIRC, it can matter in extreme cases. Less critically, writing it this way will also save a string allocation.